mne/stats/permutations.py

```diff
@@ -102,9 +102,31 @@ def permutation_t_test(X, n_permutations=10000, tail=0, n_jobs=1,
     return T_obs, p_values, H0
 
 
-def _bootstrap_ci(arr, ci=.95, n_bootstraps=2000, stat_fun='mean',
-                  random_state=None):
-    """Get confidence intervals from non-parametric bootstrap."""
+def bootstrap_ci(arr, ci=.95, n_bootstraps=2000, stat_fun='mean',
+                 random_state=None):
+    """Get confidence intervals from non-parametric bootstrap.
+
+    Parameters
+    ----------
+    arr : ndarray, shape (n_samples, ...)
+        The input data on which to calculate the confidence interval.
+    ci : float
+        Level of the confidence interval between 0 and 1.
+    n_bootstraps : int
+        Number of bootstraps
+    stat_fun : str | callable
+        Can be "mean", "median", or a callable operating along `axis=0`.
+    random_state : int | float | array_like | None
+        The seed at which to initialize the bootstrap.
+
+    Returns
+    -------
+    cis : ndarray, shape (2, ...)
+        Containing the lower boundary of the CI at `cis[0, ...]` and the upper
+        boundary of the CI at `cis[1, ...]`.
+
+
+    """
```

Comment:
we should  also review the code below for improvements.
One of the nice features of rng.choice is that it can directly generate the bootstrapped data without  the need to create indices as intermediate step. Perhaps worth updating.

Reply:
Let's do it in a follow-up PR